lib/pluginmanager/update.rb

```diff
@@ -61,7 +65,9 @@ def plugins_to_update(previous_gem_specs_map)
     if update_all?
       previous_gem_specs_map.values.map{|spec| spec.name}
     else
-      not_installed = plugins_arg.select{|plugin| !previous_gem_specs_map.has_key?(plugin.downcase)}
+      # If the plugins isn't available in the gemspec or in 
```

Comment:
I would update/add, ```or is in the gemfile dfined with a local path```, as it's now  it took me a while to understand that you don't update the plugins if:

* Not in the gemspec
* Defined with local path in the gemfile.